kedro-datasets/kedro_datasets_experimental/langchain/_anthropic.py

```diff
@@ -0,0 +1,75 @@
+"""Defines an interface to common Anthropic models."""
+
+from typing import Any, NoReturn
+
+from kedro.io import AbstractDataset, DatasetError
+from langchain_anthropic import ChatAnthropic
+
+
+class ChatAnthropicDataset(AbstractDataset[None, ChatAnthropic]):
+    """``ChatAnthropicDataset`` loads a ChatAnthropic `langchain <https://python.langchain.com/>`_ model.
+
+    Example usage for the :doc:`YAML API <kedro:data/data_catalog_yaml_examples>`:
+
+    catalog.yml:
+
+    .. code-block:: yaml
+       claude_instant_1:
+         type: langchain.ChatAnthropicDataset
+         kwargs:
+           model: "claude-instant-1"
+           temperature: 0.0
+         credentials: anthropic
+
+
+    credentials.yml:
+
+    .. code-block:: yaml
+       anthropic:
+         anthropic_api_url: <anthropic-api-base>
+         anthropic_api_key: <anthropic-api-key>
+
+    Example usage for the
+    `Python API <https://kedro.readthedocs.io/en/stable/data/\
+    advanced_data_catalog_usage.html>`_:
+
+    .. code-block:: python
+        >>> from kedro_datasets_experimental.langchain import ChatAnthropicDataset
+        >>> llm = ChatAnthropicDataset(
+        ...     credentials={
+        ...         "anthropic_api_url": "xxx"
+        ...         "anthropic_api_key": "xxx",
+        ...     },
+        ...     kwargs={
+        ...         "model": "claude-instant-1",
+        ...         "temperature": 0,
```

Comment:
```suggestion
        ...         "temperature": 0.0,
```

I find it a little bit easier when it is explicitly shown if the parameter is float.